### ISSUE TITLE
Chromedriver doesn't come with mac32 version anymore and fails install on mac

### ISSUE THREAD

**Issue by pplewa** [giggio/node-chromedriver#58]
```
Downloading http://chromedriver.storage.googleapis.com/2.23/chromedriver_mac32.zip
Saving to /var/folders/mm/l3hgkhp572s82td4ycfvfnk0kkcf1y/T/chromedriver/chromedriver_mac32.zip
Receiving...
ChromeDriver installation failed undefined
```
Should probably be changed to mac64: http://chromedriver.storage.googleapis.com/index.html?path=2.23/

already in PR: https://github.com/giggio/node-chromedriver/pull/57

**Comment by emmajam** [comment]
+ 1

**Comment by giggio** [comment]
Working version published to NPM as version `2.23.1`
